mentorship iosTests/ProfileViewModelTests.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ProfileViewModelTests.swift
+//  Created on 10/07/20
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+import XCTest
+@testable import mentorship_ios
+
+class ProfileViewModelTests: XCTestCase {
+    let profileVM = ProfileViewModel()
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+    
+    func testSaveAndGetProfile() {
+        //prepare sample data to use in saveProfile
+        let sampleProfileData = ProfileModel.ProfileData(id: 100, email: "sampleTestEmail")
+        
+        //save sample data
+        profileVM.saveProfile(profile: sampleProfileData)
+        
+        //get profile data
+        let getProfileData = profileVM.getProfile()
+        
+        //Test if save and get function works properly
+        //sample data saved and data received using get should be equal.
+        XCTAssertEqual(sampleProfileData, getProfileData)
+    }
+    
+    func testEditProfileDataReceived() {
+        //prepare sample data
+        let sampleData = ProfileModel.ProfileData(id: 100, name: nil, username: "username", email: "test@abc.com")
+        
+        //save sample data to profile
+        //reason: getEditProfileData uses getProfile function to get stored data. Hence it will get this sample data now.
+        profileVM.saveProfile(profile: sampleData)
+        
+        // get edit profile data
+        let editProfileData = profileVM.getEditProfileData()
+        
+        //id and email should be unchanged
+        XCTAssertEqual(editProfileData.id, sampleData.id)
+        XCTAssertEqual(editProfileData.email, sampleData.email)
+        
+        //name should be empty string (""). nil values are converted to allow for safe force unwrapping
+        XCTAssertEqual(editProfileData.name, "")
```

Comment:
You should have multiple samples in such testcases for values which cover: 
success cases, failure cases, edge cases etc. 

For eg: 
Say for name: 
You should test a string value, a numerical only value, nil value etc. 

Not saying you need to here. It completely depends on what kinds of values are allowed or disallowed to be stored by your data variables.

Reply:
Yeah. I've tried to include different cases while initialising the sample data through different properties (name = nil, email = "email"), etc.
I see you approved these changes earlier, should I update the PR or is it okay?